src/Sylius/Bundle/AdminBundle/Resources/views/Zone/Action/createZone.html.twig

```diff
@@ -0,0 +1,14 @@
+<div class="ui labeled icon top right floating dropdown button primary link">
+    <i class="plus icon"></i>
+    <span class="text">{{ 'sylius.ui.create'|trans }}</span>
+    <div class="menu">
+        <div class="header">
+            <i class="world icon"></i>
+            {{ 'sylius.ui.type'|trans }}
+        </div>
+        <div class="divider"></div>
+        <a class="item" href="{{ path(grid.requestConfiguration.getRouteName('create'), { 'type': 'country' }) }}"><i class="icon flag"></i>{{ 'sylius.form.zone.types.country'|trans }}</a>
+        <a class="item" href="{{ path(grid.requestConfiguration.getRouteName('create'), { 'type': 'province' }) }}"><i class="icon marker"></i>{{ 'sylius.form.zone.types.province'|trans }}</a>
+        <a class="item" href="{{ path(grid.requestConfiguration.getRouteName('create'), { 'type': 'zone' }) }}"><i class="icon location arrow"></i>{{ 'sylius.form.zone.types.zone'|trans }}</a>
```

Comment:
I am pretty sure we can't find good icons for these, so remove them for these 3 links. (the planet next to "type" text looks good)